data/mods/Magiclysm/monstergroups.json

```
@@ -189,7 +189,13 @@
     "monsters": [
       { "monster": "mon_orc_warrior", "freq": 100, "cost_multiplier": 5 },
       { "monster": "mon_orc_archer", "freq": 25, "cost_multiplier": 7 },
-      { "monster": "mon_orc_blood_warrior", "freq": 15, "cost_multiplier": 5 }
+      { "monster": "mon_orc_blood_warrior", "freq": 15, "cost_multiplier": 5 },
+      {
+        "type": "monstergroup",
+        "name": "GROUP_WORKSHOP",
+        "default": "mon_workshop_elf",
+        "monster": [ { "monster": "mon_workshop_elf", "freq": 100, "cost_multiplier": 2 } ]
```

I think you want this
```suggestion
        "monsters": [ { "monster": "mon_workshop_elf", "freq": 100, "cost_multiplier": 2 } ]
```